source/sass/wagtail/blocks.scss

```diff
@@ -97,3 +97,12 @@
 .share-button-blog-title-container {
   margin-bottom: -2em;
 }
+
+// Fixes horizontal overflow for Image Text blocks
+.tw-container.image_text-block {
+  .streamfield-content {
+    .tw-row {
+      margin-right: calc(-1rem + 1px);
+    }
+  }
+}
```

Comment:
There's an [ongoing epic](https://github.com/MozillaFoundation/foundation.mozilla.org/issues/7045) to move our codebase away from Bootstrap and Sass. So when possible, we'll use tailwind classes instead of adding in more scss / css.

Think we can try the same rule via a tailwind class in the template file instead `/networkapi/wagtailpages/templates/wagtailpages/blocks/image_text.html`?

Reply:
Thanks Rob, I was trying to figure out a better place to implement it while discussing the fix approach. I think the block template is definitely the way to go.